samples/boards/reel_board/mesh_badge/src/periphs.c

```diff
@@ -93,7 +93,7 @@ static void motion_timeout(struct k_work *work)
 	printk("power save\n");
 
 	if (!mesh_is_initialized()) {
-		k_delayed_work_submit(&motion_work, MOTION_TIMEOUT);
+		k_work_reschedule(&motion_work, MOTION_TIMEOUT);
```

Comment:
I think this too could be `k_work_schedule()`, in case a motion event was detected after the work was queued but before the work thread got to it.